feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDayTab.kt

```diff
@@ -65,10 +65,13 @@ internal fun SessionDayTab(
                 Text(
                     text = "${5 + index}",
                     style = TextStyle(
-                        textAlign = TextAlign.Center,
                         color = MaterialTheme.colorScheme.onSecondaryContainer,
                         fontSize = 24.sp,
-                        fontWeight = FontWeight(500)
+                        fontWeight = FontWeight.W500,
```

Comment:
Looks great! But there is a typography theme in the app. And it looks like the design is using it. So how about using the theme?
style = MaterialTheme.typography.headlineSmall
https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/DroidKaigi-2022-Conference-App?node-id=448%3A2009
![image](https://user-images.githubusercontent.com/1386930/190848670-83164bfd-58d7-44b0-8e2b-9b5c27c328c0.png)
https://github.com/DroidKaigi/conference-app-2022/blob/5158457fb933531d8018cd0338c6dc19d3dd1ca2/core-designsystem/src/androidMain/kotlin/io/github/droidkaigi/confsched2022/designsystem/theme/Type.kt#L68

Reply:
Sorry... I forget `headlineSmall` typography in the app.
use font where  base on `headlineSmall`.

Reply:
I fixed.
Please confirm the contents of the correction. 🙏

https://github.com/DroidKaigi/conference-app-2022/pull/540/commits/520dd0fea597acf54e41492719cbc394aa717acc